content/techniques/http-module.md

```diff
@@ -147,3 +147,31 @@ export class CatsService {
   }
 }
 ```
+
+#### Full example
+
+Since the return of `HttpService` is an Observable, we can use `rxjs` `firstValueFrom` or `lastValueFrom` to retrieve the data of the request in the form of a promise.
```

Comment:
```suggestion
Since the return value of the `HttpService` methods is an Observable, we can use `rxjs` - `firstValueFrom` or `lastValueFrom` to retrieve the data of the request in the form of a promise.
```